docs/en/engines/table-engines/mergetree-family/mergetree.md

```diff
@@ -951,15 +955,20 @@ Optional parameters:
 -   `single_read_retries` — Number of retry attempts in case of connection drop during read. Default value is `4`.
 -   `min_bytes_for_seek` — Minimal number of bytes to use seek operation instead of sequential read. Default value is `1 Mb`.
 -   `metadata_path` — Path on local FS to store metadata files for S3. Default value is `/var/lib/clickhouse/disks/<disk_name>/`.
--   `cache_enabled` — Allows to cache mark and index files on local FS. Default value is `true`.
--   `cache_path` — Path on local FS where to store cached mark and index files. Default value is `/var/lib/clickhouse/disks/<disk_name>/cache/`.
```

Comment:
this is correct configuration for version before 22.8 and the new configuration which is now showed here does not work with them. May be add some note about that? May be just a link where the old and new configuration already fully explained here https://clickhouse.com/docs/en/operations/storing-data/#using-local-cache?

Reply:
Thanks very much @kssenii !